templates/MMM-NHL.njk

```diff
@@ -94,7 +94,7 @@
                         {% if config.showLogos %}
                             <td>
                                 <img class="icon {{ "no-color" if not config.colored }}"
-                                    src="https://www-league.nhlstatic.com/images/logos/teams-20212022-dark/{{ series.teams.away.id }}.svg"/>
+                                    src="https://assets.nhle.com/logos/nhl/svg/{{games[index].teams.away.short}}_dark.svg"/>
```

Comment:
These two should be `series.teams.[away|home].short`, not `games[index]`, fyi.

Reply:
@parnic good catch